packages/@aws-cdk/aws-servicecatalog/test/portfolio.test.ts

```diff
@@ -0,0 +1,216 @@
+import '@aws-cdk/assert-internal/jest';
+import { AccountRootPrincipal, Group, Role, User } from '@aws-cdk/aws-iam';
```

Comment:
Use qualified imports for everything except `@aws-cdk/core`:

```suggestion
import * as iam from '@aws-cdk/aws-iam';
```

Reply:
Still not addressed.